llvm/test/MC/Sparc/sparc-ctrl-instructions.s

```diff
@@ -5,23 +5,44 @@
         ! CHECK:              !   fixup A - offset: 0, value: foo, kind: fixup_sparc_call30
         call foo
 
+        ! CHECK: call foo, 0  ! encoding: [0b01AAAAAA,A,A,A]
+        ! CHECK:              !   fixup A - offset: 0, value: foo, kind: fixup_sparc_call30
+        call foo, 0
```

Comment:
Can the immediate be an expression involving symbols?

Reply:
It looks like the current implementation does allow symbols though?
I'm not sure if it is worth issuing an error in this case given the obscurity of the feature, but it may be something to consider.

Reply:
> Do I need to write a custom parser to prevent that?

I'll leave that up to you :)